extension/docs/tdigest.md

```diff
@@ -90,21 +91,96 @@ FROM high_temp;
 (4 rows)
 ```
 
+## Example Using TimeScale Continuous Aggregates (tdigest-cagg-example)
+Timescale [continuous aggregates](https://docs.timescale.com/latest/using-timescaledb/continuous-aggregates)
+provide an easy way to keep a tdigest up to date as more data is added to a table.  The following example
```

Comment:
do we want to add make the types code ie `tdigest` rather than tdigest (seems like we do with a few other types like `DOUBLE PRECISION` elsewhere?